target/linux/ipq806x/patches-5.4/083-ipq8064-dtsi-additions.patch

```diff
@@ -67,7 +67,7 @@
 +	};
 +
 +	opp_table0: opp_table0 {
-+		compatible = "operating-points-v2-qcom-cpu";
++		compatible = "operating-points-v2-kyro-cpu";
```

Comment:
I read "kryo" in the patch, not kyro ...
Can you provide some details on why that naming was changed in the first place, and now is going back to kryo again ...?

Reply:
And why does the upstream commit:
```
-	ret = of_device_is_compatible(np, "operating-points-v2-kryo-cpu");
+	ret = of_device_is_compatible(np, "operating-points-v2-qcom-cpu");
```

which is removed here?

Reply:
The first one is a typo... 
The name reverted is for compatible reason
(We can't change compatible name of already pushed driver)

Reply:
Then please state it in the patch. Currently, you create the impression that this is an upstream patch, while it actually is an altered upstream patch. I shouldn't need to compare manually with upstream to find that out; do you want me to check every patch line-by-line for whether it's _really_ upstream?

I'd rather prefer to take the upstream patch as is and then add locally required adjustments in a separate patch, so it immediately obvious what's upstream and what's not (where that is possible considering backporting as well).

Reply:
Patch upstream is this
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/cpufreq/qcom-cpufreq-nvmem.c?id=a8811ec764f95a04ba82f6f457e28c5e9e36e36b
and right after there is
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/cpufreq/qcom-cpufreq-nvmem.c?id=2dea651680cea1f3a29925de51002f33d1f55711

I combined them together since it looks like a waste to have the second patch that do this change. (only a revert of the compatible)

The only other patch that is changed from the upstream version is 
- pci patchset that lacks of one include the in 8th patch 
- usb dwc3 patch that has a small fix (same thing a waste to have a second patch that fix one line)

If you want i can put restore this and add the 2 missing patch that are combined

Reply:
@adrianschmutzler  I added the deviation from the upstream patch in the patch description. Is it good now?

Reply:
As you have seen, this is an unnecessary reason for confusion. Why merge these particular patches and still keep the illusion of them being "real" upstream patches? If we locally mess with the patches again, then the whole "replace local patches by upstream thing" is just useless waste of time. For me, being an upstream patch implies that I don't have to think about it again.
So, please include the patches in their upstream version unless backporting needs adjustments. For local adjustments, create separate patches.

Reply:
Ok sorry for the mess
I will follow this path